client/layout/masterbar/publish.jsx

```diff
@@ -94,7 +94,8 @@ class MasterbarItemNew extends React.Component {
 						position={ this.getPopoverPosition() }
 					/>
 				</MasterbarItem>
-				<AsyncLoad require="layout/masterbar/drafts" />
+
+				<AsyncLoad require="layout/masterbar/drafts" placeholder={ null } />
```

Comment:
This hides a loading placeholder that would otherwise appear in the top bar. This is not related to the original issue though.

Reply:
If you have time do you mind adding a quick shot of that to the summary?

Reply:
Haha, I originally put it there but then figured it would make more sense here. I'll add it back.

Reply:
A heads-up that this fix was deployed with https://github.com/Automattic/wp-calypso/pull/19558.